src/pages/serverStatus/ServerStatus.jsx

```diff
@@ -119,9 +119,9 @@ export default function ServerStatus() {
               )}`}
             </Text>
             <Text>
-              {`ACM version: ${get(
+              {`Sage version: ${get(
```

Comment:
This specific Text component should be internationalized.
We might be able to leverage the "select format" to use something like `"Sage version: {hasVersion, select, true {{version}} other {Unknown}}"`
https://formatjs.io/docs/core-concepts/icu-syntax/#select-format